navjson/navjson.go

```diff
@@ -0,0 +1,58 @@
+// Package navjson provides a `nav.json` parser and validator.
+package navjson
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// NavFile represends a file/path association in a `nav.json` document.
+type NavFile struct {
+	Path  string    `json:"path"`
```

Comment:
should have caught this in the typescript interface you pasted in slack, but there is an optional `name` property, see - https://github.com/dequelabs/nav-json-validator/pull/1/files#diff-fd3e5f048287068c58af787eee92da0dR29

Reply:
Done